projects/train/pyproject.toml

```diff
@@ -8,7 +8,7 @@ license = "MIT"
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12" #^3.8,<3.12"
 lightning = "^2.2.1"
-"ml4gw" = {path = "../../ml4gw", develop = true}
+ml4gw = "^0.5"
```

Comment:
">=0.5"

Reply:
I think maybe `>=0.4` ? The issue we ran into was some conflicts with `aframe` libs, so in case other projects have this as a dep would be nice to have some wiggle room

Reply:
I've done `>=0.4`, let me know if that works @deepchatterjeeligo